source/common/upstream/upstream_impl.cc

```diff
@@ -85,6 +113,8 @@ getSourceAddressFnFromBindConfig(const std::string& cluster_name,
           "and additional_source_addresses fields",
           cluster_name.empty() ? "Bootstrap" : fmt::format("Cluster {}", cluster_name)));
     }
+    ENVOY_LOG_MISC(warn, "The `additional_source_addresses` field is already deprecated by "
```

Comment:
This can be deleted. This is already handled by the deprecated field tooling.

Reply:
got it